src/kedro_neptune/__init__.py

```diff
@@ -434,11 +434,14 @@ def __init__(self):
 
     @hook_impl
     def after_catalog_created(self, catalog: DataCatalog) -> None:
-        self._run_id = hashlib.md5(str(time.time()).encode()).hexdigest()
+        self._run_id = (
+            os.environ.get("NEPTUNE_CUSTOM_RUN_ID")
```

Comment:
Let's replace the hard-coded string here and in the other line with a constant `CUSTOM_RUN_ID_ENV_NAME`:
`from neptune.envs import CUSTOM_RUN_ID_ENV_NAME`